src/lib.rs

```diff
@@ -401,6 +401,11 @@
 //! - The `serialize` feature, enabled by default, provides convenience functions for serializing and deserializing with
 //! [serde](https://serde.rs/).
 //!
+//! - The `internal` feature, disabled by default, exposes functions for setting and getting
+//!   internal values not available in the default API. These functions are intended for use in
+//!   by higher-level cryptographic protocols that need access to these raw values and are able to
```

Comment:
No, I mean higher-level. The use-case here is for opaque-ke, which needs access to the underlying group elements to perform additional lower-level checks (instead of the wrapper structs BlindedElement and EvaluationElement).